iot/manager/manager.go

```diff
@@ -110,14 +117,21 @@ func getRegistry(projectID string, region string, registryID string) (*cloudiot.
 	if err != nil {
 		return nil, err
 	}
-	// [END iot_get_iam]
+	// [END iot_get_registry]
+
+	fmt.Fprintf(w, "Got registry:\n")
+	fmt.Fprintf(w, "\tID: %s\n", response.Id)
+	fmt.Fprintf(w, "\tHTTP: %s\n", response.HttpConfig.HttpEnabledState)
+	fmt.Fprintf(w, "\tMQTT: %s\n", response.MqttConfig.MqttEnabledState)
+	fmt.Fprintf(w, "\tName: %s\n", response.Name)
 
 	return response, err
 }
 
+// [START iot_get_iam_policy]
+
 // getRegistryIam gets the IAM policy for a device registry.
-func getRegistryIam(projectID string, region string, registryID string) (*cloudiot.Policy, error) {
-	// [START iot_get_iam_policy]
+func getRegistryIam(w io.Writer, projectID string, region string, registryID string) (*cloudiot.Policy, error) {
```

Comment:
s/Iam/IAM/

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Got it. Some responses don't use same-case initialisms, like `response.Id` but I'll change it for everything else